var/spack/repos/builtin/packages/gptune/package.py

```diff
@@ -17,8 +17,8 @@ class Gptune(CMakePackage):
     maintainers = ['liuyangzhuan']
 
     version('master', branch='master')
-
     variant('app', default=False, description='Build all HPC application examples')
+    variant('openmpi', default=True, description='MPI spawning-based interface')
```

Comment:
Does this package depend explicitly on `openmpi` only?

Reply:
I have changed the variant to "mpispawn" to clarify that openmpi is only needed if one would like to use the mpi spawning based feature in gptune.